libs/auth/src/lib/services/auth.interceptor.ts

```diff
@@ -41,7 +41,9 @@ export class AuthInterceptor implements HttpInterceptor {
   private async handleError(error: HttpErrorResponse): Promise<void> {
     switch (error.status) {
       case 403:
-        this.authService.setState(AuthState.Forbidden);
+        this.authService.setState(
+          error.error == 'not authorized by eIAM' ? AuthState.AccessForbidden : AuthState.ForbiddenResource
```

Comment:
I dont think ForbiddenResource should be an application state - it is not the whole application but a single resource.
Currently, once you get this error, your only way out is to reload the whole application.